gcp/modules/gke-cluster/main.tf

```diff
@@ -65,6 +66,6 @@ resource "random_id" "cluster_protector" {
     # * Change the value below to 'false'
     # * Destroy the cluster
     # * Change the value below back to 'true'
-    prevent_destroy = true
+    prevent_destroy = false
```

Comment:
You'll want to remove this along with the temporary `terragrunt mv` :)

Reply:
noted, thanks